### ISSUE TITLE
Import is working wrong in Windows 

### ISSUE THREAD

**Issue by lanycrost** [stephenh/ts-poet#32]
Hey there!

Because of the changes [made here](https://github.com/stephenh/ts-poet/pull/31/files). Imports working incorrect.

It will affect https://github.com/stephenh/ts-proto. So please fix it. It's already affecting us.

Here is explanation.

```proto
import "google/protobuf/timestamp.proto";
...
```

on Windows resulting path:
```typescript
import { Timestamp } from '..googleprotobuf\timestamp'; 
```

should be:
```typescript
import { Timestamp } from '../google/protobuf/timestamp';
```

**Comment by stephenh** [comment]
Fixed in #33

**Comment by lanycrost** [comment]
@stephenh when this will be released on npmjs.com? thanks

**Comment by stephenh** [comment]
Ah shoot, I forgot this project isn't auto-released; I just pushed up 4.14.0. Thanks!
